README.md

```diff
@@ -107,13 +107,12 @@ specific language governing permissions and limitations under the License.
 [ci-badge]: https://travis-ci.org/linkerd/linkerd2.svg?branch=master
 [cncf]: https://www.cncf.io/
 [coc]: https://github.com/linkerd/linkerd/wiki/Linkerd-code-of-conduct
-[conduit-announce]: https://groups.google.com/forum/#!forum/conduit-announce
-[conduit-demo]: https://conduit.io/getting-started/#install-the-demo-app
-[conduit-dev]: https://groups.google.com/forum/#!forum/conduit-dev
-[conduit-inject]: https://conduit.io/adding-your-service/
-[conduit-docs]: https://conduit.io/docs/
-[conduit-users]: https://groups.google.com/forum/#!forum/conduit-users
-<!-- [examples]: https://github.com/runconduit/conduit-examples -->
+[linkerd-announce]: https://groups.google.com/forum/#!forum/conduit-announce
+[linkerd-demo]: https://conduit.io/getting-started/#install-the-demo-app
+[linkerd-dev]: https://groups.google.com/forum/#!forum/conduit-dev
```

Comment:
I guess we'll need to eventually migrate mailing lists to `linkerd2-dev` or use the current ones... (our current linkerd one is `linkerd-devel`)